src/index.js

```diff
@@ -154,8 +163,8 @@ export default class StepWizard extends PureComponent {
     /** Render */
     render() {
         const props = {
-            currentStep: this.state.activeStep + 1,
-            totalSteps: this.props.children.filter(el => el).length,
+            currentStep: this.currentStep,
+            totalSteps: this.totalSteps,
```

Comment:
Do these need to be invoked here?

Reply:
Getters don't need to be invoked with `()`, they're automatically executed just by calling them

Reply:
Good to know, thanks for adding this!